docs: DOC-1770: 4.6.18 Release Notes (#6250)
spectrocloud/librarium: docs/docs-content/release-notes/release-notes.md

<ReleaseNotesVersions />

## March 28, 2025 - Release 4.6.18

### Bug Fixes

- Fixed an issue where users could not connect to Edge hosts using
  [remote shell](../clusters/edge/cluster-management/remote-shell.md) in VerteX SaaS environments.
- Fixed an issue that prevented host machines from automatically rebooting after installing the
  [Palette agent](../deployment-modes/agent-mode/install-agent-host.md).
- Fixed an issue that prevented [AWS](../clusters/public-cloud/aws/aws.md) clusters with tags containing certain special
  characters from being provisioned or deleted.
- Fixed an issue where [Edge](../clusters/edge/edge.md) native clusters remained stuck in the bootstrapping phase during
  deployment.

### Security Notices

- On March 24, 2025, a security vulnerability regarding certain versions of
  [ingress-nginx](https://github.com/kubernetes/ingress-nginx) was reported. The vulnerable versions were used in

>   Palette's and VerteX's management planes and were also available as packs for workload clusters. On March 26, 2025,

Reviewer: ```suggestion
  Palette's and VerteX's management planes and were also available as packs for workload clusters. 
  On March 26, 2025,
```
I would make this a new paragraph :) what do you think?

>   all managed Palette and VerteX management planes were patched and upgraded to a secure version, `1.11.5`. On March 28,

Reviewer: ```suggestion
  all managed Palette and VerteX deployments were patched and upgraded to a secure version, `1.11.5`. On March 28,
```
or instances

Author: Incorporated this but had to move it to a different suggestion block. Apparently git cannot combine multiple suggestions that are on the same line? Weird.

>   all managed Palette and VerteX management planes were patched and upgraded to a secure version, `1.11.5`. On March 28,

Reviewer: ```suggestion
  all managed Palette and VerteX management planes were patched and the affected component was upgraded to a secure version, `1.11.5`. On March 28,
```
maybe?

  2025, connected Palette [Enterprise](../enterprise-version/enterprise-version.md) and [VerteX](../vertex/vertex.md)

>   versions 4.4 - 4.6 were patched. Currently, airgapped Palette Enterprise and VerteX installations must manually update

Reviewer: ```suggestion
  versions 4.4 - 4.6 were patched. Currently, airgapped Palette Enterprise and VerteX installations must manually upgrade
```
